content/5_Plat/DP_Bitmasks.mdx

```diff
@@ -176,7 +176,49 @@ You can often use this to solve subtasks.
 
 ## Solution
 
-<IncompleteSection />
+<Spoiler title =  "Solution">
+
+Let our dp states be defined as $dp[S][i]$ being the amount of routes that visit all the cities in the subset $S$ and end at city $i$. The transitions will then be:
+$$
+dp[S][i] = \sum_{x \in adj[i]} dp[S/i][x] \text{ if $x \in S$}
+$$
+where $S/i$ is the subset $S$ without city $i$.
```

Comment:
We usually use a backslash (and curly braces for single elements) to denote [set difference](https://mathworld.wolfram.com/SetDifference.html)

Reply:
Should use $S \setminus \{i\}$

Reply:
er I forgot the backslashes for the curly brackets:

$S \setminus \{i\}$